rendermodules/em_montage_qc/rough_align_qc.py

```diff
@@ -0,0 +1,455 @@
+
+import renderapi
+import numpy as np 
+from math import pi
+from shapely.geometry import Polygon
+from shapely.ops import cascaded_union
+import multiprocessing as mp 
+import json
+import requests
+from functools import partial
+
+import tempfile
+import seaborn as sns
+
+from descartes.patch import PolygonPatch
```

Comment:
I'm not sure where the descartes dependency comes from, but it seems to install with render-modules.  Still, it is good to list it explicitly in the requirements file.

Reply:
added